containerized_ml_models/api/io.py

```diff
@@ -0,0 +1,8 @@
+from typing import Any
+import pickle as pkl
+
+
+def load_model(path: str) -> Any:
```

Comment:
Raczej nie powinno się używać Any (chyba że naprawdę dużo typów może zwrócić funkcja). Możesz sprawdzić typ tego modelu printując type(model)

Reply:
Function which returns pickle can return everything. Therefore I think that usage of Any type is justified.

Reply:
Then why do you use XGBoost type when using function in api.py?

Reply:
Ok, you are right